$NetBSD: patch-aa,v 1.5 2009/11/14 23:40:17 mwdavies Exp $

Add patch for CVE-2007-4400 for Konversation to stop escape song names
properly in the media script.

--- data/scripts/media.orig	2009-11-12 18:24:50.000000000 +1300
+++ data/scripts/media
@@ -414,6 +414,7 @@ def playing(playerList, mode=None):
     for i in playerList:
         s=i.get(mode)
         if s:
+            s=re.replace(r'[\r\n]', ' ', s)
             tell([IRC_SERVER, TARGET, s], 'say' )
             return 1
     return 0
